docs/source/installed_packages.rst

```diff
@@ -0,0 +1,114 @@
+.. _installed-packages:
+
+Using Installed Packages
+========================
+
+`PEP 561 <https://www.python.org/dev/peps/pep-0561/>`_ specifies how to mark
+a package as supporting type checking. Below is a summary of how to create
```

Comment:
I'd suggest adding some more background information here for those users that have minimal idea of how packaging in Python works. The goal would be that most users won't need to actually read PEP 561. Some ideas (maybe add a new section with background information or include these in the introduction):

* Write something like "a package *installed using pip, for example* ..." to make it more explicit what sort of packages we are talking about (the term 'package' is an overloaded term).
* Explain what "supporting type checking" means and why it's useful. This can mean that the package has inline type annotations that mypy can use to type check code that imports the package, or that the package ships stub files that mypy can use.
* Also contrast this to typeshed stubs and discuss which stubs take precedence in case stubs exist both in typeshed and in an installed package.
* Add note about the possibility of stubs being shipped in a separate stubs package (which might not be owned by the author of the original package).